File: ruby_tests/FizzBuzz.rb
#!/usr/bin/ruby

for i in 1..150
	if i % 15 == 0
		puts "FizzBuzz"
	elsif i % 5 == 0
		puts "Buzz"
	elsif i % 3 == 0
		puts "Fizz"
	else
		puts i
	end
end

File: perl_tests/FizzBuzz.pl
#!/usr/bin/perl
use strict;
use warnings;

for (my $i = 0; $i < 100; $i++) {
	if ($i % 15 == 0) {
		print "FizzBuzz\n";
	} elsif ($i % 5 == 0) {
		print "Buzz\n";
	} elsif ($i % 3 == 0) {
		print "Fizz\n";
	} else {
		print $i . "\n";
	}
}
